pyproject.toml

```diff
@@ -47,13 +47,12 @@ marshmallow-sqlalchemy = "0.24.2"
 werkzeug = "^1.0.0"
 html2text = "^2020.1.16"
 pycryptodome = "3.9.8"
+hubspot-api-client = "^6.0.0"
 pcdcutils = {git = "https://github.com/chicagopcdc/pcdcutils.git", rev = "0.1.2"}
 userportaldatamodel = {git = "https://github.com/chicagopcdc/userPortalDataModel.git", rev = "1.4.1"}
 # pcdcutils = {git = "https://github.com/chicagopcdc/pcdcutils.git", rev = "pcdc_dev"}
-# hubspotclient = {git = "https://github.com/chicagopcdc/hubspotclient.git", rev = "0.1.0"}
 # userportaldatamodel = {git = "https://github.com/chicagopcdc/userPortalDataModel.git", rev = "pcdc_dev"}
-hubspotclient = {git = "https://github.com/chicagopcdc/hubspotclient.git", rev = "1.0.1"}
-pcdc-aws-client = {git = "https://github.com/chicagopcdc/pcdc-aws-client.git", rev = "0.3.3"}
+pcdc-aws-client = {git = "https://github.com/chicagopcdc/pcdc-aws-client", rev = "1.0.1"}
```

Comment:
Did you make sure everywhere else in the app, something is imported from this package has been updated?